.github/lighthouse/puppeteer-script.js

```diff
@@ -0,0 +1,8 @@
+module.exports = async (browser, context) => {
```

Comment:
Could we use a real script, and run it as a separate example workflow?

Reply:
That would need quite a bit edits of page, which is in the examples. If needed I can do it ;)

Reply:
I meant, maybe some basic example, like visit google.com, search, and audit the search results page, to ensure, that the script is actually triggered.